match/lib/match/change_password.rb

```diff
@@ -1,31 +1,44 @@
 require_relative 'module'
-require_relative 'encrypt'
-require_relative 'git_helper'
+
+require_relative 'storage'
+require_relative 'encryption'
 
 module Match
   # These functions should only be used while in (UI.) interactive mode
   class ChangePassword
-    def self.update(params: nil, from: nil, to: nil)
+    def self.update(params: nil)
       ensure_ui_interactive
-      to ||= ChangePassword.ask_password(message: "New passphrase for Git Repo: ", confirm: false)
-      from ||= ChangePassword.ask_password(message: "Old passphrase for Git Repo: ", confirm: true)
-      GitHelper.clear_changes
-      workspace = GitHelper.clone(params[:git_url],
-                                  params[:shallow_clone],
-                                  manual_password: from,
-                                  skip_docs: params[:skip_docs],
-                                  branch: params[:git_branch],
-                                  git_full_name: params[:git_full_name],
-                                  git_user_email: params[:git_user_email],
-                                  clone_branch_directly: params[:clone_branch_directly])
-      Encrypt.new.clear_password(params[:git_url])
-      Encrypt.new.store_password(params[:git_url], to)
+
+      to ||= ChangePassword.ask_password(message: "New passphrase for Git Repo: ", confirm: true)
```

Comment:
@milch Can you explain more what you mean here? I think I might be a bit confused 😖

Reply:
Of course, sorry about that. I'm just saying that "git repo" is not accurate anymore, and all references outside of the git storage should rename it to some other consistent value. If you want to be fancy you could add a method to the Storage interface (e.g. `human_readable_name`) and use that when referencing the storage:

```
ChangePassword.ask_password(message: "New passphrase for #{storage.human_readable_name}:")
```

Would then print this if I were using git storage: 

```
New passphrase for Git Repo:
```

Or if I used, say, a theoretical "Simple SFTP Storage" (SSS for short) that called its version of repositories "Canisters" then it would print 

```
New passphrase for SSS Canister:
```